schema-engine/sql-migration-tests/tests/migrations/sql/vitess.rs

```diff
@@ -1,4 +1,4 @@
-use migration_engine_tests::test_api::*;
+use sql_migration_tests::test_api::*;
```

Comment:
PR that includes this test into the module tree here: https://github.com/prisma/prisma-engines/pull/4023